frontend/src/lib/components/Badges/Badge.svelte

```diff
@@ -15,13 +15,13 @@
 
   interface Props {
     variant?: BadgeVariant;
-    icon?: IconString | undefined;
-    hoverIcon?: IconString | undefined;
+    icon?: IconString;
+    hoverIcon?: IconString;
     outline?: boolean;
     children?: Snippet;
   }
 
-  let {
+  const {
     variant = 'badge-neutral',
     icon = undefined,
     hoverIcon = undefined,
```

Comment:
I wasn't going to mention this, but you were so thorough that I didn't see any others (I only looked at the files you touched)
```suggestion
    icon,
    hoverIcon,
```

Reply:
Fixed in commit d957532e.